app/templates/_package.json

```diff
@@ -2,14 +2,14 @@
     "name": "<%= _.slugify(appname) %>",
     "version": "0.0.1",
     "dependencies": {
-        "deployd": "~0.6.10"
+        "deployd": "*"
```

Comment:
It can be dangerous to use any version because of breaking changes.
We could use `^0.6` so will update the version until there's a breaking change (since deployd and grunt use semver).
Same for grunt & co.

What do you think?